lms/static/js/spec/staff_debug_actions_spec.js

```diff
@@ -14,6 +14,12 @@ define(['backbone', 'jquery', 'js/staff_debug_actions'],
                 });
             });
 
+            describe('sanitize_string', function () {
+                it('escapes escapable characters in a string', function () {
+                    expect(StaffDebug.sanitized_string('P1:test_loc')).toBe('P1\\:test_loc');
+                });
+            });
+
```

Comment:
I second that, might be smart to improve on this test and do checking on that whole set. Otherwise :+1: